vsintegration/Vsix/VisualFSharpFull/Source.extension.vsixmanifest

```diff
@@ -5,7 +5,9 @@
     <Identity Id="VisualFSharp" Version="15.4.1.0" Language="en-US" Publisher="Microsoft.VisualFSharpTools" />
     <DisplayName>Visual F# Tools</DisplayName>
     <Description xml:space="preserve">Deploy Visual F# Tools templates to Visual Studio</Description>
-    <ShortcutPath>..\CommonExtensions\Microsoft\FSharp</ShortcutPath>
+    <PackageId>Microsoft.FSharp.VSIX.Full.Core</PackageId>
+    <ExtensionInstallationRoot>CommonExtensions</ExtensionInstallationRoot>
+    <ExtensionInstallationFolder>Microsoft\FSharp</ExtensionInstallationFolder>
```

Comment:
These VSIXes both go to Microsoft\FSharp, which is not allowed.  They need to go to different installation folders.